.github/workflows/test.yml

```diff
@@ -1,12 +1,16 @@
-name: Test
+name: Core Repo Tests
 
 on:
   push:
     branches-ignore:
     - 'release/*'
   pull_request:
 env:
-  CONTRIB_REPO_SHA: 5c9e043d6921550d82668788e3758a733fb11cb8
+  # Set variable to 'master' if your change will not affect Contrib.
+  # Otherwise, set variable to the commit of your branch on
+  # opentelemetry-python-contrib which is compatible with these Core repo
+  # changes.
+  CONTRIB_REPO_SHA: 9398616add7071085a9a8448899a7de178b84bbf
```

Comment:
It's probably better to wait for https://github.com/open-telemetry/opentelemetry-python-contrib/pull/131 so that we can change this to `master` expecting `master` to be Green most of the time

Reply:
It's been merged! We can use `master` now! 🥳